flagging_site/README.md

```diff
@@ -0,0 +1,10 @@
+# flagging_site Content 
+
+  - blueprints: contains files that are modular applications added to main flask application
+  - data: contains files that retrieves and cleans the hobolink and usgs data
+  - static: contains images and css/style sheet
+  - templates: contains the html pages that define the web page structure
+  - __init__.py: required to treat directory as a package
+  - app.py: main flask application
+  - config.py: file holding the config options for flask and 
+  - vault.zip: contains credentials necessary to accessing hoblink and usgs api 
```

Comment:
Very good! And you definitely understand the app very well, which is no small feat as there's a lot going on. 🎉 On line 9 it looks like you forgot to finish the sentence.